docs/usage/configuration-options.md

```diff
@@ -235,6 +235,11 @@ Use the `automergeSchedule` option to define times of week or month during which
 The default value for `automergeSchedule` is "at any time", which functions the same as setting a `null` schedule.
 To configure this option refer to [`schedule`](#schedule) as the syntax is the same.
 
+<!-- prettier-ignore -->
+!!! warning
+    When `platformAutomerge` is enabled, Renovate enqueues the platform PR automerge at time of creation, so the schedule specified in `automergeSchedule` cannot be followed.
+    As a workaround, set `platformAutomerge: false`, configure `schedule` to control PR creation, and enforce successful pipelines before merging on your platform.
```

Comment:
```suggestion
    If it's essential that automerging only happens within the specific `automergeSchedule` time window, then you need to set `platformAutomerge` to `false` and instead rely on Renovate's automerge instead of the platform one.
```

Reply:
Looks a nice change! But you don't think it's worth mentioning anything about successful pipelines (i.e. `Require status checks to pass before merging` in GitHub and `Merge checks → Pipelines must succeed` in GitLab)? This is one of the big things you get "for free" when using `platformAutomerge`

Reply:
you always get it also without platform merge because renovate will wait for all status checks to succeed.

Reply:
Right, Renovate won't merge if any status checks are pending or failed - this is the default behavior

Reply:
Ah, I see, thanks for the feedback :-) Then I don't have any other remarks.

I think it was the documentation on [platformAutomerge](https://docs.renovatebot.com/configuration-options/#platformautomerge) that made me believe status checks was not obeyed:

> If you use the default platformAutomerge=true then you should enable your Git hosting platform's capabilities to enforce test passing before PR merge. If you don't do this, the platform might merge Renovate PRs even if the repository's tests haven't started, are in still in progress, or possibly even when they have failed.